.ci/run-container

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
`run-container` -> would be nice to keep `ci` in the name, `run_container` sounds a bit too generic

Reply:
ok. renamed to `run-container-ci`.